src/Rules/Resources/ConditionDescriptions.resx

```diff
@@ -119,130 +119,167 @@
   </resheader>
   <data name="AllChildren" xml:space="preserve">
     <value>AllChildren({0})</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="And" xml:space="preserve">
     <value>{0} and {1}</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="AnyChild" xml:space="preserve">
     <value>AnyChild({0})</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="BoundingRectangleCompletelyObscuresContainer" xml:space="preserve">
     <value>BoundingRectangle completely obscures container</value>
+    <comment>The name of a rule used by the automated accessibility scanner.</comment>
   </data>
   <data name="ChildrenExist" xml:space="preserve">
     <value>children exist</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="ChildCount" xml:space="preserve">
     <value>CountChildren({0})</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="CustomLandmark" xml:space="preserve">
     <value>custom landmark</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="DescendantCount" xml:space="preserve">
     <value>DescendantCount({0})</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="False" xml:space="preserve">
     <value>False</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="FormLandmark" xml:space="preserve">
     <value>form landmark</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="HeadingLevel" xml:space="preserve">
     <value>heading level</value>
+    <comment>Do not translate: a token used by the Axe.Windows domain-specific language for accessibility descriptions.</comment>
   </data>
   <data name="IsContentElement" xml:space="preserve">
     <value>IsContentElement</value>
+    <comment>The name of a UI Automation property that should not be translated.</comment>
```

Comment:
Consistency question: Most of the non-translated strings start with "Do not localize:". Seven of them phrase it as "should not be translated". Would it make sense to have them all follow the same pattern?

Reply:
It does: looks like I missed some of these while attempting to uniformize.